swan-lake/learn/keeping-ballerina-up-to-date.md

```diff
@@ -56,26 +55,13 @@ Ballerina distribution is a term that we use to refer to jBallerina and nBalleri
 
 ### Release channels
 
-Ballerina distributions are released on two different release channels at the moment: patch releases and minor releases. Both these channels distribute stable versions. Ballerina yet does not have a release channel for nightly builds that give you access to the latest perhaps unstable features.
-
-Ballerina distribution releases strictly follow [SemVer](https://semver.org/) with major.minor.patch version numbers.
+Ballerina distributions of the Swan Lake channel are currently released as preview releases. 
 
-#### Patch release channel
-
-This channel gives you access to the patch releases of Ballerina distributions that contain bug fixes and fixes for critical stability and security related issues. These releases are strictly time-bound and happen every two weeks. Occasionally,  you would see on-demand patch releases as well.
-
-*Example patch releases: 1.0.6, 1.1.5, 1.1.10*
-
-#### Minor release channel
-
-This channel gives you access to feature releases of Ballerina distributions. Ballerina programs that you’ve written today should continue to work on these minor releases. There will be four minor releases a year. You will get access to a minor release on the 3rd Wednesday of the 3rd month of every quarter.
-
-*Example minor releases: 1.1.0, 1.2.0, 1.3.0*
+For example: Swan Lake Preview 1
 
 #### Release maintenance
 
-- We maintain a minor release 1.x.0 by issuing a series of patch releases 1.x.y. The maintenance of a particular minor release stops when there are two newer minor releases available.
-- In other words, patch releases for 1.x.0 stop when jBallerina 1.(x+2).0 is released. E.g., when 1.2 is available, maintaining 1.0.0 will be stopped.
+The maintenance of a particular preview release stops when the stable version of it is available. For example, the preview releases of Swan Lake will stop when the stable Swan Lake GA version is released.
```

Comment:
In preview releases we don't have anything for maintenance. Once GA released its a stable release. Until 1.2.x will be maintained